src/app/routes/api/v3/admin/forms/__tests__/admin-forms.settings.routes.spec.ts

```diff
@@ -265,4 +265,179 @@ describe('admin-form.settings.routes', () => {
       })
     })
   })
+
+  describe('DELETE /forms/:formId/logic/:logicId', () => {
```

Comment:
should this test block be in a separate `admin-form.logic.routes.spec.ts` file?

Reply:
edited